docs/known-issues.md

```diff
@@ -0,0 +1,33 @@
+![OpenFin Web Starter](../assets/openfin-web-starter.png)
+
+> **_:information_source: OpenFin:_** [OpenFin](https://www.openfin.co/) libraries are a commercial product and this repo is for evaluation purposes. Use of the OpenFin npm packages is only granted pursuant to a license from OpenFin. Please [**contact us**](https://www.openfin.co/contact/) if you would like to request a developer evaluation key or to discuss a production license.
+
+## OpenFin Core Web - Known Issues
+
+### [0.38.55](https://www.npmjs.com/package/@openfin/core-web/v/0.38.55)
+
+#### Layout containing a section with more than one view
+
+If you have more than one tab in a section you set reorderEnabled to false as it will be fully supported in a future release.
```

Comment:
grammar nit:
`... one tab in a section, you must set ...`

Reply:
Update pushed.